.github/workflows/gh-actions-artifacts-snapshots.yml

```diff
@@ -0,0 +1,80 @@
+name: Build example projects and publish them to GitHub Actions Artifacts
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  stdlib:
+    runs-on: ubuntu-latest
+    if: github.repository == 'Kotlin/dokka'
+    steps:
+      - name: Checkout dokka
+        uses: actions/checkout@v3
+        with:
+          submodules: true
+      - uses: actions/setup-java@v3
+        with:
+          distribution: 'zulu'
+          java-version: 12
+      - name: Document stdlib
+        uses: gradle/gradle-build-action@v2
+        with:
+          arguments: :integration-tests:gradle:integrationTest --tests org.jetbrains.dokka.it.gradle.kotlin.StdlibGradleIntegrationTest --stacktrace
+        env:
+          DOKKA_IT_AWS_PATH: /home/runner/work/dokka/stdlib      
```

Comment:
We need to rename `S3Project` & `DOKKA_IT_AWS_PATH`?
https://github.com/Kotlin/dokka/blob/56beaa318c603c0ada0cf97454f06fbb44175917/integration-tests/src/main/kotlin/org/jetbrains/dokka/it/S3Project.kt#L6-L16